apollo-adminservice/src/assembly/assembly-descriptor.xml

```diff
@@ -46,7 +46,11 @@
 			<directory>target/classes</directory>
 			<outputDirectory>/config</outputDirectory>
 			<includes>
+				<!-- sort by alphabet -->
 				<include>application-github.properties</include>
+				<include>application-h2.properties</include>
+				<include>application-mysql.properties</include>
+				<include>application-postgre.properties</include>
```

Comment:
Currently, there is no need for the user to change database specific configurations, so I think we don't need to copy them the config folder?

```suggestion

```

Reply:
How about we put `application-h2.properties`, `application-mysql.properties` and `application-postgre.properties` to `apollo-common/src/main/resources` to avoid duplications?